WordPress/gutenberg
# Using addQueryArgs to generate Manage All Reusable Blocks link

## Description
Using `addQueryArgs` method  from `@wordpress/url` instead of hand coding 'Manage All Reusable Blocks' link.

## How has this been tested?
1. Open 'Reusable' panel in 'Add Block' menu
2. Check 'Manage All Reusable Blocks' link is redirecting to `edit.php?post_type=wp_block`.
2. Open 'More' menu (top right ...)
2. Check 'Manage All Reusable Blocks' link is redirecting to `edit.php?post_type=wp_block`.

## Screenshots 

## Types of changes
Reuse an existing method.

## Checklist:
